contestants.json

```diff
@@ -255,6 +263,6 @@
 	   "facebook" : "https://facebook.com/utkarshsingh369",
 	   "github" : "https://github.com/utkarshsingh99",
 	   "twitter" : ""
-      }  
+      },
```

Comment:
no `,` here.

Reply:
now there is no conflict

Reply:
But make requested changes. They are not ok.